src/lib/backpack/jpeg-thumbnail.js

```diff
@@ -6,17 +6,24 @@ const jpegThumbnail = dataUrl => new Promise((resolve, reject) => {
 
         const maxDimension = 96; // 3x the maximum displayed size of 32px
 
-        if (image.height > image.width) {
-            canvas.height = maxDimension;
-            canvas.width = (maxDimension / image.height) * image.width;
+        if (image.height < 1 || image.width < 1) {
+            canvas.width = canvas.height = maxDimension;
+            // drawImage can fail if image height/width is less than 1
+            // Use blank image; the costume is too small to render anyway
+            ctx.fillStyle = 'white'; // Create white background, since jpeg doesn't have transparency
+            ctx.fillRect(0, 0, maxDimension, maxDimension);
```

Comment:
This is slightly more maintainable (in the case that we ever change canvas.width/height).

```suggestion
            ctx.fillRect(0, 0, canvas.width, canvas.height);
```